qt/res/app.organicmaps.desktop.metainfo.xml

```diff
@@ -107,6 +107,23 @@
   </screenshots>
 
   <releases>
+   <release version="2022.05.24-2" date="2022-05-24">
+     <description>
+       <ul>
+         <li>Allow using emulated storage for downloads</li>
+         <li>Fixed GPS search on start</li>
+         <li>Fixed accidental position jumps near WiFi</li>
```

Comment:
These 3 lines are relevant only for Android builds.

Reply:
Removed